packages/netlify-cms-core/src/components/Editor/Editor.js

```diff
@@ -156,10 +156,10 @@ class Editor extends React.Component {
        */
       const values = deserializeValues(entry.get('data'), fields);
       const deserializedEntry = entry.set('data', values);
-      const fieldsMetaData = nextProps.entryDraft && nextProps.entryDraft.get('fieldsMetaData');
+      const fieldsMetaData = this.props.entryDraft && this.props.entryDraft.get('fieldsMetaData');
       this.createDraft(deserializedEntry, fieldsMetaData);
```

Comment:
This seems proper to be in `componentDidUpdate` as well, but maybe it's not? Any thoughts?